README.md

```diff
@@ -6,6 +6,22 @@
 
 The project uses poetry for Python to create an isolated environment and manage package dependencies. To prepare your system, ensure you have an official distribution of Python version 3.7+ and install Poetry using one of the following commands (as instructed by the [poetry documentation](https://python-poetry.org/docs/#system-requirements)):
 
+
+### Running with Docker
+
+Setup is simple with Docker (and Docker compose)! Simply run:
+```bash
+docker compose up development
```

Comment:
Definitely personal preference, but I recommend using `--build` by default with docker compose, as far too often I've gotten myself mixed up when I've made a change to e.g. the Dockerfile and then not been able to see it working